app/src/main/java/org/mozilla/focus/search/SearchEngineParser.java

```diff
@@ -99,6 +99,9 @@ private static void readUrl(XmlPullParser parser, SearchEngine searchEngine) thr
             if (tag.equals("Param")) {
                 final String name = parser.getAttributeValue(null, "name");
                 final String value = parser.getAttributeValue(null, "value");
+                if (value.equals("{searchTerms}")) {
+                    searchEngine.searchTermsParamName = name;
+                }
```

Comment:
I guess in theory we could get this value from a SearchEngine already by looping over the query parameters of the first resultsUris?